app/src/components/Registration/Dropdown.tsx

```diff
@@ -6,54 +6,59 @@ import {
     FormControl,
     Select,
 } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
 
-const DropdownSelect = (props: {
-    name: string
+const useStyles = makeStyles((theme) => ({
+    formControl: {
+        margin: theme.spacing(0),
+        minWidth: 200,
+    },
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+}))
+
+interface Props {
     value: string
     onChange: (event: object) => void
-    placeholder: string
     id: string
-    itemValue: string
-    className: string
-    htmlFor: string
     helperText: string
     options: Array<string>
-}) => {
-    const {
-        name,
-        value,
-        onChange,
-        placeholder,
-        id,
-        itemValue,
-        className,
-        htmlFor,
-        helperText,
-        options,
-    } = props
+    label: string
+    labelId: string
+}
+
+const DropdownSelect = (props: Props) => {
+    const classes = useStyles()
+    const { value, onChange, id, helperText, options, label } = props
 
     return (
         <>
-            <form className={className}>
-                <FormControl>
-                    <InputLabel htmlFor={htmlFor}>{placeholder}</InputLabel>
+            <form>
+                <FormControl
+                    variant={`outlined`}
+                    className={classes.formControl}
+                >
+                    <InputLabel id={id}>{label}</InputLabel>
                     <Select
+                        labelId={id}
                         value={value}
                         onChange={onChange}
                         inputProps={{
                             name,
                             id,
                         }}
                     >
+                        <MenuItem value="">
+                            <em>None</em>
+                        </MenuItem>
                         {options.map((menuItem: string) => {
                             return (
-                                <MenuItem value={itemValue}>
```

Comment:
this might be where an ID might be mapped into the value prop instead of the display text